### PR TITLE
Completed javascript-primitives-and-variables Exercise - Bingle Padilla

### PR DESCRIPTION
**Pull request by codewithbingle**
javascript-primitives-and-variables

### CODE REVIEW

File: javascript-primitives-and-variables/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+var isCool = true;
+var totalPets = 1;
+var unknown = null;
+var never;
+
+console.log('value of fullName: ' + fullName);
+console.log('value of isCool: ' + isCool);
+console.log('value of totalPets: ' + totalPets);
+console.log('typeof fullName: ' + typeof 'Bingle Padilla');
+console.log('typeof isCool: ' + typeof true);
```
Comment (Zhiwutian) [inline javascript-primitives-and-variables/main.js:11]: you want to use the variables here, not the values

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
make sure you are using the variables in the console logs!

**Comment by codewithbingle** [comment]
javascript-primitives-and-variables-update2

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**